assets/js/atomic/blocks/product-elements/price/types.ts

```diff
@@ -0,0 +1,5 @@
+export interface BlockAttributes {
```

Comment:
I also noticed that `isDescendentOfQueryLoop` is missing from this interface which is present [here](https://github.com/woocommerce/woocommerce-blocks/blob/64684d4a9d6068aaf8189e5633691c9c08089d20/assets/js/atomic/blocks/product-elements/price/attributes.ts#L11).  May I know if there is any specific reason to omit it?

Reply:
@nielslange Tagging you here in case you missed this comment. 🙂

Reply:
Thanks for your ping. Initially, I haven't added `isDescendentOfQueryLoop` as I thought it was a leftover of an older codebase because this attribute didn't appear in block.tsx. I just added the attribute accordingly.